package/boot/uboot-rockchip/Makefile

```diff
@@ -27,19 +27,40 @@ define U-Boot/rockpro64-rk3399
     pine64_rockpro64
   DEPENDS:=+PACKAGE_u-boot-rockpro64-rk3399:arm-trusted-firmware-rockchip
   PKG_BUILD_DEPENDS:=arm-trusted-firmware-rockchip
+  ATF:=rk3399_bl31.elf
+endef
+
+define U-Boot/nanopi-r2s-rk3328
```

Comment:
Consider moving that up due to alphabetic sorting (similarly for the UBOOT_TARGETS list below).

Reply:
I've sorted this by SoC now.